Parse/Internal/PlatformHooks/Phone/PlatformHooks.Phone.cs

```diff
@@ -150,17 +150,12 @@ partial class PlatformHooks : IPlatformHooks {
 
     public string DeviceTimeZone {
       get {
-        // We need the system string to be in english so we'll have the proper key in our lookup table.
-        var culture = Thread.CurrentThread.CurrentCulture;
-        Thread.CurrentThread.CurrentCulture = CultureInfo.InvariantCulture;
-        string windowsName = TimeZoneInfo.Local.StandardName;
-        Thread.CurrentThread.CurrentCulture = culture;
-
-        if (ParseInstallation.TimeZoneNameMap.ContainsKey(windowsName)) {
-          return ParseInstallation.TimeZoneNameMap[windowsName];
-        } else {
-          return null;
-        }
+        TimeSpan utcOffset = TimeZoneInfo.Local.BaseUtcOffset;
+        return String.Format("GMT{0}{1}:{2:d2}",
```

Comment:
Yep, you're correct, except our backend only supports `Etc/GMT+1` (no minutes)... So maybe back to the drawing board. Damn.